commcare_connect/opportunity/forms.py

```diff
@@ -303,3 +303,36 @@ def __init__(self, *args, **kwargs):
             self.fields["deliver_units"].initial = [
                 deliver_unit.pk for deliver_unit in self.instance.deliver_units.all()
             ]
+
+
+class SendMessageMobileUsersForm(forms.Form):
+    title = forms.CharField(
+        empty_value="Notification from CommCare Connect",
+        required=False,
+        widget=forms.TextInput(attrs={"placeholder": "Notification from CommCare Connect"}),
```

Comment:
you can remove the placeholder since it gets set by default as "empty value"

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/240/commits/6a97662b8cb04970ec2855fc71861bb6d491d60f